app/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,14 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="Schedule_day">
```

Comment:
There is some built in styles from AppCompat. See https://gist.github.com/Aracem/922d460b73168676ca21 parents. I hope they will be useful.